controller/openebs/reconciler.go

```diff
@@ -276,6 +276,7 @@ func (p *Planner) init() error {
 		p.setPoliciesDefaultsIfNotSet,
 		p.setAnalyticsDefaultsIfNotSet,
 		p.removeDisabledManifests,
+		p.checkCSISupport,
```

Comment:
Instead of calling it as a function from here, can we just set this as CSI defaults i.e. if not supported then disable it from there only so that it can automatically be handled by disableManifests function and we can have a log also at a higher verbose level maybe which will just say that CSI will be not installed as the version is not supported if it is enabled.

Reply:
Yes, I have updated it.